protoc-gen-elm/main.go

```diff
@@ -231,12 +257,15 @@ func (fg *FileGenerator) GenerateComments(inFile *descriptor.FileDescriptorProto
 	fg.P("-- source file: %s", inFile.GetName())
 }
 
-func (fg *FileGenerator) GenerateImports() {
+func (fg *FileGenerator) GenerateImports(includeDictImport bool) {
```

Comment:
Maybe rename this to GenerateBaseImports and then keep the conditional logic in the caller method only?

Reply:
Done